Dockerfile.webapi

```diff
@@ -0,0 +1,20 @@
+FROM python:3.8-alpine
+
+RUN apk --no-cache --update add bash inotify-tools curl
+
+WORKDIR /usr/src/app
+
+COPY requirements.txt ./
+RUN pip install -r requirements.txt
+
+RUN mkdir -p /events
+WORKDIR /events
+VOLUME [ "/events" ]
```

Comment:
I think, you do not need the /events directory, do you?

Reply:
not required. removed this part from Dockerfile.